### DISCUSSION TITLE
How to remove emoji which we dont want to be shown?

### DISCUSSION THREAD

**Discussion by masdedy** [Aghajari/AXEmojiView#13] [Q&A]
Dear Brother,
shokran about your good job. 
Please kindly advice how to remove the emoji which we dont want it to be shown?

**Comment by Aghajari** [comment]
Hi, currently there is no method for this issue but you can create your own EmojiProvider and ignore the emojis which you want to hide..

**Comment by Aghajari** [comment]
Hi @masdedy ,
since 1.5.0:
Now you can filter the emojis that you don't want to show:
```java
AXEmojiManager.filterEmojis(Arrays.asList("😀", "😃", "😄", "😁"));
```
